workspace/cmd/api2/api2.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"log"
+	"net/http"
+
+	apiv2 "github.com/bufbuild/buf-examples/workspace/gen/proto/go/api/v2"
+	"github.com/bufbuild/buf-examples/workspace/gen/proto/go/api/v2/apiv2connect"
+	"github.com/bufbuild/connect-go"
+	"golang.org/x/net/http2"
+	"golang.org/x/net/http2/h2c"
+	emptypb "google.golang.org/protobuf/types/known/emptypb"
+)
+
+var ErrorNeedsImplementation = errors.New("needs implementation")
```

Comment:
Thanks, I changed it to be a proper `connect.NewError(...)`

But kept the more fleshed out example of the service which I think is worthwhile because our code does not in general always auto import correctly.

It's a very odd step between code generation with `buf generate proto` and then importing it.